packages/world/src/ObjectType.sol

```diff
@@ -1155,24 +1155,51 @@ library ObjectTypeLib {
     coords[0] = baseCoord;
 
     for (uint256 i = 0; i < schemaCoords.length; i++) {
-      require(isDirectionSupported(self, direction), "Direction not supported");
-      coords[i + 1] = baseCoord + schemaCoords[i].rotate(direction);
+      require(isOrientationSupported(self, orientation), "Orientation not supported");
```

Comment:
we should move this outisde the if statement because rn its not getting called for single voxels

Reply:
nice catch, will add a test